Makefile.core.mk

```diff
@@ -31,6 +31,7 @@ generate-config:
 	@(cd prow/config/cmd; go run generate.go write)
 	@rm -fr prow/cluster/jobs/istio-private/*/*.gen.yaml
 	@go run prow/genjobs/main.go --configs=./prow/config/istio-private_jobs
+	@go run prow/genjobs/main.go --configs=./prow/config/istio-private_jobs
```

Comment:
why duplicate this?

Reply:
Fixed. I was going to remove the `istio-private.envoy` file. Decided not to remove it.